README.md

```diff
@@ -37,7 +37,7 @@ previous cluster managers such as [kops][kops] and
 
 This provider's versions are compatible with the following versions of Cluster API:
 
-|                                    | v1alpha3 (v0.3) | v1alpha4 (v0.4) | v1beta1 (v1.1) |
+|                                    | v1alpha3 (v0.3) | v1alpha4 (v0.4) | v1beta1 (v1.2) |
```

Comment:
Do we want to add a further column for `v1beta1 (v1.2)` or just replace it like you've done it?
I'm just asking because we have both `v1alpha4` releases (i.e., `v0.4` and `v0.5`).

Reply:
I think we should add another column as v0.5 and v0.6 only supports CAPI v1.1